tests/test_output.py

```diff
@@ -1178,3 +1178,34 @@ def test_save_models_to_current_directory():
     eq_(learner1.model_type, learner2.model_type)
     eq_(learner1.model_params, learner2.model_params)
     eq_(learner1.model_kwargs, learner2.model_kwargs)
+
+
+def test_get_feature_names():
```

Comment:
This is a good test but it also needs to cover the case where the feature selector doesn't remove anything. An easy change would be to turn this into a test generator (`check_get_feature_names()`) that takes a boolean argument of whether or not to force the feature selection and then check both cases. Take a look at the other `check_X/test_X` function pairs in `test_output.py`.

Reply:
addressed in #[4f34a5d](https://github.com/EducationalTestingService/skll/commit/4f34a5db7bbe5809760f8937d947c63d0cf44343)